doc/source/deployment/configure.rst

```diff
@@ -190,6 +190,16 @@ For example:
 
    socok8s_ext_vip: "10.10.10.10"
 
+Set Up Retry Files Save Path
+----------------------------
+
+Before beginning deployment, a path must be specified where Ansible retry files
```

Comment:
It's actually not a _must_. Those retry files are not necessary and if ansible fails to create them (after a failure during execution), it's just one more error, I would not say a critical one.